app/src/main/java/com/dodo/flashcards/presentation/viewCardsScreen/ViewCardsViewModel.kt

```diff
@@ -0,0 +1,42 @@
+package com.dodo.flashcards.presentation.viewCardsScreen
+
+import com.dodo.flashcards.architecture.BaseRoutingViewModel
+import com.dodo.flashcards.domain.models.flashcard.Flashcard
+import com.dodo.flashcards.domain.models.flashcard.IFlashcard
+import com.dodo.flashcards.presentation.MainDestination
+import dagger.hilt.android.lifecycle.HiltViewModel
+import javax.inject.Inject
+import com.dodo.flashcards.presentation.viewCardsScreen.ViewCardsViewEvent.SwipedAnyDirection
+
+@HiltViewModel
+class ViewCardsViewModel @Inject constructor(
+) : BaseRoutingViewModel<ViewCardsViewState, ViewCardsViewEvent, MainDestination>() {
+
+	//placeholder list construction
+	var cards = listOf<IFlashcard>()
+
+	init {
+		//init cards with firestore or mock impl here
+		cards = listOf<IFlashcard>(Flashcard(), Flashcard())
```

Comment:
See my comment about making the implementation a data class, that way for our mocking here you can just go `Flashcard("this is my front", "this is my back")`